docs/semgrep-assistant/overview.md

```diff
@@ -83,6 +83,10 @@ Autofixes are also available in Semgrep AppSec Platform's **Findings page** unde
 
 The finding's details also include a link to the PR or MR with the autofix, so you can go directly to the PR or MR to commit the autofix.
 
+:::info 
+If there are a large number of new issues found in a given scan, Assistant auto-triage and autofix may not run on every issue.
```

Comment:
This is just my curiosity: if this were to happen, what would be the next steps for the user? Manually triggering auto-triage and autofix?

Reply:
@Sly1029 might need to confirm, but I'm not exactly sure. They wouldn't get the PR comments AFAIK if it was a PR scan (which is where this most often applies).

Reply:
@khorne3 Do you think we should add a CTA of sorts here? I'm open to that if there is a good action item.

Reply:
I think it is fine for now; we can change if it becomes a problem (or if we think it will)